renovate-config.json

```diff
@@ -24,6 +24,7 @@
     "rollbackPrs": true,
     "packageRules": [
         {"matchDepTypes": ["require"], "rangeStrategy": "widen"},
+        {"matchDepTypes": ["require-dev"], "rangeStrategy": "bump"},
```

Comment:
And here, modify the default rangeStrategy instead of adding another package rule.